### PR TITLE
feat(tui): add Russian (`ru`) locale

### PR DESCRIPTION
**Pull request by fujiapple852**

### CODE REVIEW

File: crates/trippy-tui/locales/app.yml [hunk 1]
```diff
@@ -6,730 +6,835 @@ trippy:
...
   zh: "帮助"
+  ru: "помощь"
 header_settings:
   en: "settings"
   fr: "paramètres"
   tr: "ayarlar"
   it: "impostazioni"
   pt: "configurações"
   zh: "设置"
+  ru: "hастройки"
```
Comment (histrio) [inline crates/trippy-tui/locales/app.yml:233]: `настройки`

**Comment by fujiapple852** [comment]
Thanks @histrio !

File: crates/trippy-tui/locales/app.yml [hunk 2]
```diff
@@ -6,730 +6,835 @@ trippy:
...
   zh: "扩展"
+  ru: "Расширение"
 help_tagline:
   en: "A network diagnostic tool"
   fr: "Un outil de diagnostic réseau"
   tr: "Bir ağ analiz aracı"
   it: "Uno strumento diagnostico di rete"
   pt: "Uma ferramenta de diagnóstico de rede"
   zh: "网络诊断工具"
+  ru: "Сетевой диагностический инструмент"
```
Comment (histrio) [inline crates/trippy-tui/locales/app.yml:465]: rather
`Инструмент диагностики сети`

File: crates/trippy-tui/locales/app.yml [hunk 3]
```diff
@@ -6,730 +6,835 @@ trippy:
...
   zh: "多个 GeoIp 位置"
+  ru: "Несколько местоположений GeoIp для прыжка"
 kilometer:
   en: "km"
   fr: "km"
   tr: "km"
   it: "km"
   pt: "km"
   zh: "公里"
+  ru: "km"
```
Comment (histrio) [inline crates/trippy-tui/locales/app.yml:537]: `км`

File: crates/trippy-tui/locales/app.yml [hunk 4]
```diff
@@ -6,730 +6,835 @@ trippy:
...
   zh: "最后"
+  ru: "Посл"
 column_avg:
   en: "Avg"
   fr: "Moyenne"
   tr: "Ort"
   it: "Media"
   pt: "Média"
   zh: "平均"
+  ru: "Ср"
```
Comment (histrio) [inline crates/trippy-tui/locales/app.yml:721]: rather
`Сред`

File: crates/trippy-tui/locales/app.yml [hunk 5]
```diff
@@ -6,730 +6,835 @@ trippy:
...
+  ru: "МаксДжитр"
 column_jint:
   en: "Jint"
   fr: "GigInt"
   tr: "Jint"
   it: "Jint"
   pt: "Jint"
   zh: "抖动间隔"
+  ru: "ИнтДжитр"
 column_sprt:
```
Comment (histrio) [inline crates/trippy-tui/locales/app.yml:786]: Not clear context. What's sprt/dprt here?

**Review by histrio — CHANGES_REQUESTED** [review]

File: crates/trippy-tui/locales/app.yml (hunk 5 above)
Comment (fujiapple852) [reply crates/trippy-tui/locales/app.yml:786]: Source and destination ports. Keep in mind these are column names and horizontal space is at a premium so short/abbreviated words are best.

File: crates/trippy-tui/locales/app.yml [hunk 6]
```diff
@@ -6,730 +6,835 @@ trippy:
...
   en: "Sprt"
   fr: "Psrc"
   tr: "Sprt"
   it: "Sprt"
   zh: "源端"
+  ru: "Порт"
```
Comment (histrio) [inline crates/trippy-tui/locales/app.yml:792]: `Исх`

File: crates/trippy-tui/locales/app.yml [hunk 7]
```diff
@@ -6,730 +6,835 @@ trippy:
...
 column_dprt:
   en: "Dprt"
   fr: "Pdest"
   tr: "Dprt"
   it: "Dprt"
   pt: "Dprt"
   zh: "目标"
+  ru: "Порт"
```
Comment (histrio) [inline crates/trippy-tui/locales/app.yml:800]: `Назн`

File: crates/trippy-tui/locales/app.yml (hunk 4 above)
Comment (histrio) [reply crates/trippy-tui/locales/app.yml:721]: Or you can keep `Ср` if the length is critical

**Review by histrio — CHANGES_REQUESTED** [review]

**Comment by fujiapple852** [comment]
@histrio all changes applied, here is how it looks:

Screenshot 2024-11-04 at 7 36 47 PM

File: crates/trippy-tui/locales/app.yml [hunk 8]
```diff
@@ -830,11 +933,13 @@ column_nat:
...
 column_fail:
   en: "Fail"
   fr: "Échec"
   tr: "Başarısız"
   it: "Falliti"
   pt: "Falha"
   zh: "失败"
-  sv: "Misslyckades"
\ No newline at end of file
+  sv: "Misslyckades"
```
Comment (histrio) [inline crates/trippy-tui/locales/app.yml:944]: another language

**Review by histrio — CHANGES_REQUESTED** [review]

File: crates/trippy-tui/locales/app.yml (hunk 8 above)
Comment (fujiapple852) [reply crates/trippy-tui/locales/app.yml:944]: It is just the change in EOL as it is no longer the final line in the file, no worries!

**Review by histrio — APPROVED** [review]

File: crates/trippy-tui/locales/app.yml (hunk 8 above)
Comment (histrio) [reply crates/trippy-tui/locales/app.yml:944]: got it

**Review by histrio — APPROVED** [review]

**Pull request merged**